pkg/handler/hooks.go

```diff
@@ -12,6 +12,12 @@ type HookEvent struct {
 	HTTPRequest HTTPRequest
 }
 
+// HookResponse is the response after a hook is executed.
+type HookResponse struct {
+	// Updated metadata which can be set from the pre create hook and is then used instead of the initial metadata.
```

Comment:
Please add `ID` here.

Reply:
I added UpdatedID to handler.hookResponse. 

Also fine for you @Acconut ? 
We could move this field also to the proposed handler.UpdatedFileInfo for sure.